hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/KeyValueContainer.java

```diff
@@ -169,6 +169,7 @@ public void create(VolumeSet volumeSet, VolumeChoosingPolicy
           String hddsVolumeDir = containerVolume.getHddsRootDir().toString();
           // Set volume before getContainerDBFile(), because we may need the
           // volume to deduce the db file.
+          containerData.setCommittedSpace(true);
```

Comment:
Once the volume is selected, and container creation fails, we should release the committed space.